astropy/visualization/wcsaxes/core.py

```diff
@@ -197,17 +198,25 @@ def imshow(self, X, *args, **kwargs):
 
         # To check whether the image is a PIL image we can check if the data
         # has a 'getpixel' attribute - this is what Matplotlib's AxesImage does
-
         try:
             from PIL.Image import Image
-            from PIL.Image.Transform import FLIP_TOP_BOTTOM
         except ImportError:
             # We don't need to worry since PIL is not installed, so user cannot
             # have passed RGB image.
             pass
         else:
+            with warnings.catch_warnings():
+                # turn warning into exception
+                warnings.filterwarnings('error', category=DeprecationWarning)
+                try:
+                    # FLIP_TOP_BOTTOM was deprecated in Pillow 9.1.0
+                    from PIL.Image import FLIP_TOP_BOTTOM as flip_top_bottom
+                except DeprecationWarning:
+                    from PIL.Image import Transpose
+                    flip_top_bottom = Transpose.FLIP_TOP_BOTTOM
+
             if isinstance(X, Image) or hasattr(X, 'getpixel'):
-                X = X.transpose(FLIP_TOP_BOTTOM)
+                X = X.transpose(flip_top_bottom)
```

Comment:
```suggestion
                X = X.transpose(FLIP_TOP_BOTTOM)
```
Revert back to `FLIP_TOP_BOTTOM`

Reply:
What is the point changing the case here? Why not use it as PIL named it? 👍 to revert.